simpleast-core/src/main/java/com/discord/simpleast/core/simple/SimpleMarkdownRules.kt

```diff
@@ -21,7 +21,7 @@ object SimpleMarkdownRules {
   val PATTERN_UNDERLINE = Pattern.compile("^__([\\s\\S]+?)__(?!_)")
   val PATTERN_STRIKETHRU = Pattern.compile("^~~(?=\\S)([\\s\\S]*?\\S)~~")
   val PATTERN_NEWLINE = Pattern.compile("""^(?:\n *)*\n""")
-  val PATTERN_TEXT = Pattern.compile("^[\\s\\S]+?(?=[^0-9A-Za-z\\s\\u00c0-\\uffff]|\\n\\n| {2,}\\n|\\w+:\\S|$)")
+  val PATTERN_TEXT = Pattern.compile("^[\\s\\S]+?(?=[^0-9A-Za-z\\s\\u00c0-\\uffff]|\\n| {2,}\\n|\\w+:\\S|$)")
```

Comment:
Allow each newline to terminate the pattern text, and have the newline node handle consuming the rest